src/components/dialog/DialogBase.js

```diff
@@ -52,7 +53,7 @@ export const DialogBase = ({
             onEscKeyDown={onEscKeyDown}
           >
             <FocusRing>
-              <div ref={ref} data-teamleader-ui="dialog" className={dialogClassNames}>
+              <Box ref={ref} data-teamleader-ui="dialog" className={dialogClassNames} {...rest}>
```

Comment:
I'm also wondering if this could break something, the default display value for a div differs from a form

`div: display: block`
`form: display: inline-block`

Reply:
It only becomes a form element if you actually pass `element="form"` to the `Dialog` so I don't think this is an issue